### ISSUE TITLE
include parameters in autocomplete

### ISSUE THREAD

**Issue by janosimas** [joeroback/atom-clang#13]
The autocomplete is working great, very fast.
I miss the parameters are included in the completed code.

It would be a plus if I could cycle the parameters with "tab". 
:-)

**Comment by joeroback** [comment]
yep, i certainly agree. i am working on a few things now (segfault, .clang_complete and compile_commands.json support, and bug with the linter), but i consider this one very high in the nice to have category. should be fairly easy to show in the completion list.

maybe one day it would be nice to have it populate the editor like Xcode and let you tab through the arguments as your replace them, but i think we would need more support from autocomplete-plus and atom editor to get that done.
